src/js/editor/post.js

```diff
@@ -9,9 +11,24 @@ function isMarkupSection(section) {
   return section.type === MARKUP_SECTION_TYPE;
 }
 
+function isTextEditableSection(section) {
```

Comment:
Should probably just remove it if the value of the abstraction has been diminished/removed.